app/models/vm_or_template/scanning.rb

```diff
@@ -8,6 +8,12 @@
 module VmOrTemplate::Scanning
   extend ActiveSupport::Concern
 
+  # Smartstate Analysis is unsupported by default.
+  # Subclasses need to override this method if they support SSA.
+  included do
+    supports_not :smartstate_analysis, :reason => _("Smartstate Analysis is not available for VM or Template.")
```

Comment:
It looks like this generates a less informative message than the original `validate_unsupported` method defined in `app/models/vm_or_template/operations.rb`. The original message indicates the type of VM or Template for which SSA is not supported:
`is not available for #{self.class.model_suffix} VM or Template.`